docs/errors/forms.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Form errors
+description: An index of Clerk errors related to forms.
+type: reference
+---
+
+# Forms Errors
```

Comment:
```suggestion
```
Seems that this is not needed as the title will be used as header 
![CleanShot 2024-08-22 at 23 50 38@2x](https://github.com/user-attachments/assets/1f6a5755-fc1a-4cd9-a3f0-24411d398979)